ui/core/src/config.sandpit.json

```diff
@@ -2,8 +2,45 @@
   "sifAddrPrefix": "sif",
   "sifApiUrl": "https://lcd-sandpit.sifchain.finance/",
   "sifWsUrl": "wss://rpc-sandpit.sifchain.finance/websocket",
+  "sifRpcUrl": "https://rpc-sandpit.sifchain.finance/",
   "sifChainId": "sandpit",
   "web3Provider": "metamask",
   "nativeAsset": "rowan",
-  "bridgebankContractAddress": "0xa5658e85bc7AC7F0388E60E58BEd5aFB41143330"
+  "bridgebankContractAddress": "0xD67F01EdA3541BBf0D17A87BBe8B30E21A822B26",
+  "keplrChainConfig": {
+    "chainName": "Sifchain",
+    "chainId": "",
```

Comment:
this should be `sandpit`

Reply:
It is set in the `parseConfig` function based on the `sifChainId` above. Could we remove it here @59023g ?

Reply:
It's defined here, then object is updated in parseconfig.ts https://github.com/Sifchain/sifnode/pull/493/files/72ead783f2b5909bb9b91920da2bc2e468865e9a#diff-1f2cf9825b529857b03f1e86239bcc8fdd2000da15ae45a590a801cf6bb5bc6bL5

Reply:
yeah ok.